bower.json

```diff
@@ -1,15 +1,16 @@
 {
-	"name": "o-quote",
-	"main": [
-		"main.scss"
-	],
-	"ignore": [
-		"test",
-		".travis.yml"
-	],
-	"dependencies": {
-		"o-colors": "^4.0.0",
-		"o-icons": ">=4.0.0 <6",
-		"o-typography": "^5.0.0"
-	}
+  "name": "o-quote",
+  "main": [
+    "main.scss"
+  ],
+  "ignore": [
+    "test",
+    ".travis.yml"
+  ],
+  "dependencies": {
+    "o-colors": "^4.0.0",
+    "o-icons": ">=4.0.0 <6",
+    "o-typography": "^5.0.0",
+    "o-spacing": "^2.0.0"
```

Comment:
🙈

Reply:
Is the 🙈 because it's at v2? Yeah that was me. A moment of great shame and confusion.